.eslintrc.json

```diff
@@ -29,20 +29,20 @@
       ],
       "parserOptions": {
         "ecmaVersion": 2020,
-        "project": "tsconfig.json"
+        "project": ["tsconfig.json", "cypress/tsconfig.json"]
       },
       "plugins": ["@typescript-eslint", "filenames"],
       "rules": {
         "curly": ["error", "all"],
         "no-console": "off",
         "no-plusplus": "off",
         "no-useless-escape": "off",
-        "require-await": "error",
+        "require-await": "off",
```

Comment:
If you return an async fn you don't need to await it. eslint forces you to await it